test/inductor/test_torchinductor_opinfo.py

```diff
@@ -253,6 +288,109 @@ def format_op(op):
     },  # NYI: could not find kernel for aten.view.default at dispatch key DispatchKey.SparseCUDA
 }
 
+inductor_expected_failures_single_sample["xpu"] = {
+    "_upsample_bilinear2d_aa": {f16, f32, f64},
```

Comment:
@hoshibara In terms of some operations, it is surprised that we could not support some data types like `fp16`. @fengyuan14 , @xytintel  FYI